### PR TITLE
Qrexec expose

### PR DESCRIPTION
**Pull request by linse**
We don't automatically invoke gui and qrexec for -t qubes anymore.
In QubesOS 3.2 it was necessary to start default handlers for all VMs,
otherwise QubesOS would assume the VM was not working properly and kill it
after 60 seconds.  This is configurable in QubesOS 4, so now that
QubesOS 3.2 is past end-of-life, don't do it by default.
/cc @yomimono

### CODE REVIEW

File: lib/mirage.mli [hunk 1]
```diff
@@ -53,6 +53,9 @@ val qubesdb: qubesdb typ
 val default_qubesdb: qubesdb impl
 (** A default qubes database, guessed from the usual valid configurations. *)
 
+type qrexec
```
Comment (talex5) [inline lib/mirage.mli]: Some doc-comments are needed here. Most people reading this aren't going to know what qrexec (or Qubes) is.

**Comment by yomimono** [comment]
> I wonder if we should go further - do you think it's useful to keep qrexec support in the mirage tool at all? It seems like it can't select something else with the same interface for e.g. -t unix.

Hm, that's a good point.  Less being more and all, I think that's the way to go.

**Comment by yomimono** [comment]
(On further thought I think @linse suggested this too, and I had some half-baked idea about it being useful to have setup done before the unikernel starts, but after having slept on it I don't think that logic is very sound.)

**Comment by talex5** [comment]
Closing in favour of #984 - thanks!

**Pull request closed without merging**